sentry/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,82 @@
+                                                                                                     odoo.exceptions.Warning,
+                                                                                                     odoo.exceptions.except_orm``
+
+``sentry_processors``          A string of comma-separated processor classes which will be applied   ``raven.processors.SanitizePasswordsProcessor,
+                               on an event before sending it to Sentry.                              odoo.addons.sentry.logutils.SanitizeOdooCookiesProcessor``
+
+``sentry_transport``           Transport class which will be used to send events to Sentry.          ``threaded``
+                               Possible values: *threaded*: spawns an async worker for processing
+                               messages, *synchronous*: a synchronous blocking transport;
+                               *requests_threaded*: an asynchronous transport using the *requests*
+                               library; *requests_synchronous* - blocking transport using the
+                               *requests* library.
+
+``sentry_include_context``     If enabled, additional context data will be extracted from current    ``True``
+                               HTTP request and user session (if available). This has no effect
+                               for Cron jobs, as no request/session is available inside a Cron job.
+
+``sentry_release``             Explicitly define a version to be sent as the release version to
+                               Sentry. Useful in conjuntion with Sentry's "Resolve in the next
+                               release"-functionality. Also useful if your production deployment
+                               does not include any Git context from which a commit might be read.
+                               Overrides *sentry_odoo_dir*.
+
+``sentry_odoo_dir``            Absolute path to your Odoo installation directory. This is optional
+                               and will only be used to extract the Odoo Git commit, which will be
+                               sent to Sentry, to allow to distinguish between Odoo updates.
+                               Overridden by *sentry_release*
+=============================  ====================================================================  ==========================================================
+
+Other `client arguments
+<https://docs.sentry.io/clients/python/advanced/#client-arguments>`_ can be
+configured by prepending the argument name with *sentry_* in your Odoo config
+file. Currently supported additional client arguments are: ``install_sys_hook,
+include_paths, exclude_paths, machine, auto_log_stacks, capture_locals,
+string_max_length, list_max_length, site, include_versions, environment``.
+
+Example Odoo configuration
+--------------------------
```

Comment:
You should use `~` as underline here. cc/ @moylop260 (from https://github.com/OCA/maintainer-tools/issues/459)

Reply:
Hi @sbidoul 
Thanks for check it
Thank you
I will fix it